2020-03-17 17:23:10,896 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2096, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2020-03-17 17:31:03,846 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:31:57,705 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:32:54,512 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:33:18,009 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:35:17,585 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:35:52,389 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:36:27,100 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:37:17,801 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:49:04,280 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:57:13,304 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:58:55,741 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:59:40,500 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:00:29,534 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:01:30,101 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:16:45,526 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:28:27,017 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:30:32,454 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:34:22,449 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:44:14,295 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:44:18,488 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:47:40,403 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:50:10,784 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:54:22,889 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:55:27,691 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:57:09,337 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 18:58:28,547 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:00:15,876 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:04:40,049 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:04:42,251 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:04:45,249 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 191, in translate_text
    return jsonify({'text': translate(request.form['text'],
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\translate.py", line 21, in translate
    return json.loads(response.json.content.decode('utf-8-sig'))
AttributeError: 'function' object has no attribute 'content'
2020-03-17 19:05:23,632 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:05:28,756 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 191, in translate_text
    return jsonify({'text': translate(request.form['text'],
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\translate.py", line 21, in translate
    return json.loads(r.content.decode('utf-8-sig'))
AttributeError: 'list' object has no attribute 'content'
2020-03-17 19:08:37,830 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:14:24,028 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:14:27,903 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:15:57,653 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:19:42,689 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:25:55,816 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 19:28:14,509 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-18 13:20:06,968 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:74]
